src/services/accounts/AccountsConvertService.spec.ts

```diff
@@ -0,0 +1,201 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import { ApiPromise } from '@polkadot/api';
+
+import { sanitizeNumbers } from '../../sanitize';
+import { defaultMockApi } from '../test-helpers/mock';
+import { AccountsConvertService } from './AccountsConvertService';
+
+const mockApi = {
+	...defaultMockApi,
+} as unknown as ApiPromise;
+const validateService = new AccountsConvertService(mockApi);
+
+describe('Convert accounts', () => {
+	it('Should convert a Substrate accountId to a SS58 Address with scheme=ecdsa', () => {
```

Comment:
```suggestion
	it('Should convert a Substrate accountId to a SS58 Address when `scheme` equals `ecdsa`', () => {
```

Reply:
Not sure what you think but I think it's a little cleaner to keep test descriptions as complete english. Same for the tests below.

Reply:
Not a huge deal though, and will leave this up to you. It was just a thought when I was reading it. That being said there is cases in Sidecar where we do exactly what you did above with comparison symbols such as `>` and `<=` so im totally cool with keeping it the way it is if you think its best.

Reply:
I agree in changing it! I like it better :)